package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gitcoin",
+  "version": "1.0.0",
+  "description": "Gitcoin is the easiest way to monetize or incentivize work in Open Source Software",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint -c .eslintrc ./app/assets/v2/js/**/*.js",
+    "build": "webpack --config webpack.config.js --progress --colors",
+    "build-production": "webpack --config webpack.prod.config.js --progress --colors",
+    "watch": "node server.js"
```

Comment:
we don't have a `server.js` yet ! Skip it for now?

Reply:
Haha dangit!  I missed something else!  Good thing for CR 👍